src/main.js

```diff
@@ -6,10 +6,23 @@ import { CompositeDisposable, Task } from 'atom'
 // Dependencies
 // NOTE: We are not directly requiring these in order to reduce the time it
 // takes to require this file as that causes delays in Atom loading this package
-let path
-let helpers
-let workerHelpers
-let isConfigAtHomeRoot
+//
+// The functions first assigned to the dependency variables will self-assign
+// their underlying modules once called in an animation frame.
+
+/* eslint-disable no-return-assign */
+let path = () => path = require('path')
+let helpers = () => helpers = require('./helpers')
+let workerHelpers = () => workerHelpers = require('./worker-helpers')
+let isConfigAtHomeRoot = () =>
+  isConfigAtHomeRoot = require('./is-config-at-home-root')
+/* eslint-enable */
+
+// Load the dependencies when Atom next has free time.
+//
+requestAnimationFrame(() => {
```

Comment:
If we are staying with this, make it explicit that this is `window.requestAnimationFrame()`.